core/templates/services/image-local-storage.service.ts

```diff
@@ -0,0 +1,77 @@
+// Copyright 2014 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Service for managing images in localStorage.
+ */
+
+require('services/alerts.service.ts');
+require('services/image-upload-helper.service.ts');
+
+angular.module('oppia').factory('ImageLocalStorageService', [
+  'AlertsService', 'ImageUploadHelperService', function(
+      AlertsService, ImageUploadHelperService) {
+    var imagesStored = [];
+    // According to https://en.wikipedia.org/wiki/Web_storage, 5MB is the
+    // minimum limit, for all browsers, that can be stored in localStorage.
+    var imageStorageLimit = 5 * 1024 / 100;
```

Comment:
Perhaps explain the "100" in the comment.

Also is it the minimum limit per page or in total for all pages visited by the browser? Perhaps clarify in the comment.

Reply:
Done